CMakeLists.txt

```
@@ -33,11 +33,20 @@ if(CMAKE_Fortran_COMPILER_ID STREQUAL GNU)
     add_compile_options(-std=f2008ts)
   endif()
 elseif(CMAKE_Fortran_COMPILER_ID STREQUAL Intel)
-  add_compile_options(-warn declarations,general,usage,interfaces,unused)
-  if(CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 18.0)
-    add_compile_options(-stand f15)
-  else()
-    add_compile_options(-stand f18)
+  if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

I think we should use [CMAKE_HOST_SYSTEM_NAME](https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_SYSTEM_NAME.html#variable:CMAKE_HOST_SYSTEM_NAME) here in case somebody tries to cross-compile stdlib.
```suggestion
  if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows")
```